generatehtml.py

```diff
@@ -15,6 +15,18 @@ def generate_html(data_folder):
     source_files = parsed_json['source_files']
     file_obj.write('TN\n')
     for source_file in source_files:
+        if ignore_third_party is True:
+            ignore = False
+            with open('mozilla-central/tools/rewriting/ThirdPartyPaths.txt') as f:
```

Comment:
We are reading the file multiple times (once for every source_file), we should read it once at the beginning and just reuse its contents.